modules/monitoring-and-analyzing-the-health-of-your-clusters.adoc

```diff
@@ -15,9 +19,9 @@ Wait until the Compliance Operator returns the scan results. It might take a few
 
 .Procedure
 
-. In the {product-title-short} portal, go to the *Compliance (2.0) -> Cluster Compliance -> Coverage* tab.
+. In the {product-title-short} portal, go to *Compliance -> Coverage*.
 . Select a cluster to view the details of the individual scans.
-. Optional: Enter the name of the compliance check in the *Filter by keyword box* to view the status.
+. Optional: Enter the name of the profile check in the *Filter by keyword box* to view the status.
 . Optional: From the *Compliance status* drop-down list, select one or more statuses by using which you want to filter the scan details.
 +
 The following values are supported:
```

Comment:
Can we borrow the descriptions of what these statuses mean from the CO documentation? https://docs.openshift.com/container-platform/4.16/security/compliance_operator/co-concepts/compliance-operator-crd.html#compliance-check-result_compliance-crd